ice_setup/ice.py

```diff
@@ -1353,7 +1354,13 @@ def configure_remote(
 
     return destination_name
 
+def handle_ceph_deploy_ioerror(exc):
+    if exc.errno == EROFS:
+        print 'Error: Please ensure the current working directory is writable.'
+    print str(exc)
+    raise SystemExit(1)
```

Comment:
if you just raise `SystemExit` with the message it will do a non-zero status code as well